src/components/structures/auth/LanguageSelector.js

```diff
@@ -32,7 +32,8 @@ export default function LanguageSelector() {
     if (SdkConfig.get()['disable_login_language_selector']) return <div />;
 
     const LanguageDropdown = sdk.getComponent('views.elements.LanguageDropdown');
-    return <div className="mx_Login_language_div">
-        <LanguageDropdown onOptionChange={onChange} className="mx_Login_language" value={getCurrentLanguage()} />
-    </div>;
+    return <LanguageDropdown className="mx_Auth_language"
+        onOptionChange={onChange}
+        value={getCurrentLanguage()}
```

Comment:
I'm not sure what our standard is (if any), but I've noticed at least my IDE gets annoyed and tries to align these with the other properties like so:

```
<LanguageDropdown className="mx_Auth_language"
                  onOptionChange={onChange}
                  value={getCurrentLanguage()}
/>
```

Just a mention more than anything that we might want to pick one kind of format

Reply:
Ah I see, I would be fine with that style too, assuming I can find some option or extension to make my editor agree as well... Probably something to discuss for next retro.

Reply:
Have added it to the retro board, and my personal notes in case it gets lost before we retro.